mandatory/3-function-output.js

```diff
@@ -1,18 +1,20 @@
-// Add comments to explain what this function does. You're meant to use Google!
+//The getRandomNumber function will return a random integer from 0 to 9.
+
 function getRandomNumber() {
   return Math.random() * 10;
 }
```

Comment:
That is correct. The `Math.random()` function returns a number between 0 and 1(inclusive of 0, but not including 1), and then it will just multiply that value by 10 as we wrote `* 10` after the method.